components/MaterialsForm/Machines/index.js

```diff
@@ -30,7 +52,7 @@ export default function Machines() {
           key={`${index}-${newMachines[index].machinesAndDevices}`}
         >
           <Label htmlFor={`machinesAndDevices-${index}`}>
-            Maschinen und Geräte
+            <span>Maschinen</span>
```

Comment:
Why do you need this `span`?

Reply:
wanted to try something and forgot to take it away😅